<launch>
    <arg name="serial_port" default="/dev/ttyACM0" />
    <arg name="baud_rate" default="115200" />
    <node pkg="uwb" type="uwb_range_node.py" name="uwb_range_node" output="screen">
        <param name="serial_port" value="$(arg serial_port)" />
        <param name="baud_rate" value="$(arg baud_rate)" />
        <param name="uwb_topic" value="/uwb/range" />
    </node>
</launch>
